src/main/scala/com/advancedtelematic/director/http/AdminResource.scala

```diff
@@ -248,45 +236,10 @@ class AdminResource(extractNamespace: Directive1[Namespace], val keyserverClient
       (path("images") & get) {
         listInstalledImages(ns, device)
       } ~
-      // Deprecated in favor of "assignments/:deviceId" endpoint
-      pathPrefix("queue") {
```

Comment:
I find it a little annoying that this project uses *versioned* APIs: https://github.com/advancedtelematic/director/blob/master/src/main/scala/com/advancedtelematic/director/http/DirectorRoutes.scala but then decides to make API changes that will break people's tooling that depends on these APIs.

Reply:
Understood, will redo this. Thanks!